# Single tracked target information
std_msgs/Header header
uint32 track_id
geometry_msgs/Point position
geometry_msgs/Vector3 velocity
float64[36] covariance
float64 timestamp
